zh/restore-data-using-tidb-lightning.md

```diff
@@ -79,14 +81,48 @@ tikv-importer 可以在一个现有的 TiDB 集群上启用，或者在新建 Ti
               [s3]
               type = s3
               provider = AWS
-              env_auth = false
+              env_auth = true
```

Comment:
This is not gonna work, in this case, the rclone.conf is mounted to the container directly instead of env.

Reply:
If the rclone is not config in value， this will be used， and in values, the config is being commended